src-plugins/itkDataSHImage/management/vtkSphericalHarmonicSource.cxx

```diff
@@ -92,7 +92,6 @@ static void
 TranslateAndDeformShell(vtkPolyData* shell,vtkPoints* outPts,double center[3],
                         bool deform,vtkMatrix4x4* transform=0);
 
-vtkCxxRevisionMacro(vtkSphericalHarmonicSource,"$Revision: 0 $");
```

Comment:
The comment is not for this line but for line 45 (I do not know how to comment on lines which were not modified by the PR):  
const double factor = static_cast<double>(2*l+1)/(4.0*vtkMath::DoublePi())*fact;

When I tried to compile the PR to test it I had an error because of vtkMath::DoublePi() work when using vtkMath::Pi() instead. I checked with MedInria 3 they are using:
 const double factor = static_cast<double>(2*l+1)/(8.0*vtkMath::Pi())*fact;

Reply:
it's indeed strange, i did the change DoublePi -> Pi everywhere but not here

Reply:
Oh that's because i'm on linux without clang. Thx for the debug

Reply:
Done also :)